met-web/src/components/engagement/form/EngagementWidgets/EmptyOptionCard.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { MetPaper } from 'components/common';
+import { Grid } from '@mui/material';
+
+const EmptyOptionCard = () => {
```

Comment:
We don't need this, could be confusing to have it

Reply:
ok will remove